modules/gdscript/editor/gdscript_highlighter.cpp

```diff
@@ -293,7 +293,11 @@ Dictionary GDScriptSyntaxHighlighter::_get_line_syntax_highlighting_impl(int p_l
 
 			String word = str.substr(j, to - j);
 			Color col = Color();
-			if (keywords.has(word)) {
+			if (global_functions.has(word)) {
+				if (word == "assert" || word == "preload" || str[to] == '(') {
```

Comment:
Why do those need special casing here since they were already appended to the HashSet?